src/index.ts

```diff
@@ -47,10 +48,10 @@ export type TransactionOptions = {
  * */
 export async function launch(
   puppeteerLib: typeof puppeteer,
-  { args, ...rest }: LaunchOptions = {},
+  { args, metamaskLocation, ...rest }: LaunchOptions = {},
```

Comment:
allow env to override metamaskLocation this for CI purposes
also rename ti to downloadLocation or metamaskDownloadLocation

Reply:
> allow env to override metamaskLocation this for CI purposes

Somewhere in `js` developer need to use `launch` and there he can set env variables and they wish.

> also rename ti to downloadLocation or metamaskDownloadLocation

if a user manually download metamask he can set a path to that location, its is not exclusive for download